nvflare/fuel/utils/function_utils.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
What is this for? It doesn't seem related to data bus. Should be in a separate PR.